inference/environments/minimal-ubuntu20.04-py38-cpu-inference/environment.yaml

```diff
@@ -0,0 +1,26 @@
+image:
+  name: azureml/curated/minimal-ubuntu20.04-py38-cpu-inference
```

Comment:
Is this name registered with MCR?

Reply:
yes, we registered this in MCR 
https://github.com/microsoft/mcr/blob/a7be5aa1ff7778b28c38417b7c25e1b254e5ead3/teams/azureml/base/curated-public.yml#:~:text=%2D%20name%3A%20public/azureml/curated/minimal%2Dubuntu20.04%2Dpy38%2Dcpu%2Dinference